docs/user/reference/destinations.md

```diff
@@ -0,0 +1,19 @@
+# Destinations
```

Comment:
Will there be a page per reference? As a lover of monopages, this makes me sad. But I'll get over it.

We thought the existing docs were mostly reference. Will those...remain? Have their content migrated to these new pages?

Don't feel strongly, just curious about the plan.

Reply:
As mentioned in other comment, there are advantages to each concept having its own page. It can be a canonical URI, and even if we do move it, VSCode will automatically update references. But I agree it hurts spontaneous discovery / casual scrolling

Reply:
We can also always write a little script that accumulates all the little pages into a monopage for folks who like that sort of thing.

Let's move forward with the path of least resistance and edit from there.

Reply:
I think I prefer a single page that has in index and maybe a very short summary. I prefer if each concept and its description has a single page with a canonical uri.